{"timestamp":"2025-10-16T17:19:47.584682Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-16T17:19:47.585114Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/fetch_and_load_data_dag.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-16T17:19:47.614349Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:47.614471Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:47.614564Z","level":"info","event":"Current task name:fetch_and_load_data","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:47.614605Z","level":"info","event":"Dag name:fetch_load_dbt_pipeline","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:48.399373Z","level":"info","event":"ETL process starting...","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:48.399370Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":994,"error_detail":[{"exc_type":"NameError","exc_value":"name 'config' is not defined","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":82,"name":"run"},{"filename":"/opt/airflow/dags/fetch_and_load_data_dag.py","lineno":20,"name":"fetch_and_load_data"},{"filename":"/opt/airflow/src/main.py","lineno":18,"name":"main"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-16T17:19:48.400102Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:48.400289Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:48.400346Z","level":"info","event":"Task:<Task(PythonOperator): fetch_and_load_data>","logger":"task.stdout"}
{"timestamp":"2025-10-16T17:19:48.400413Z","level":"info","event":"Failure caused by name 'config' is not defined","logger":"task.stdout"}
